ci/cases/pr/C48mx500_3DVarAOWCDA.yaml

```diff
@@ -9,7 +9,7 @@ arguments:
   resdetocean: 5.0
   comroot: {{ 'RUNTESTS' | getenv }}/COMROOT
   expdir: {{ 'RUNTESTS' | getenv }}/EXPDIR
-  icsdir: {{ 'ICSDIR_ROOT' | getenv }}/C48mx500
+  icsdir: {{ 'ICSDIR_ROOT' | getenv }}/C48mx500/20240610
```

Comment:
Not for this PR, but I wonder if we could add a `ci/platforms/config.common` that specifies the IC date (e.g. `20240610`).

Reply:
I was thinking something similar, although along the lines of creating a new versions file. The versions are all the same at the moment so it would be needed if we have a new version of an existing IC set in the future. Either way, it would be a good follow-up PR to do something to make the timestamp a variable and have it set once place.